resources/documentation/stories.rst

```diff
@@ -0,0 +1,467 @@
+=======
+Stories
+=======
+
+This part contains few stories to help you face common tasks / issues seen
+while using the stack.
+
+Management and main configuration
+=================================
+
+Versioning the inventory
+------------------------
+
+It may be a good idea to version the whole /etc/bluebanquise/inventory/
```

Comment:
Rephrase with **It is a good practice**?

Reply:
Solved.